.github/fix-maven.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+MAVEN_HOME=$(./jdk-wrapper.sh ./mvnw -v |grep 'Maven home:' | awk '{ print $3 }')
```

Comment:
nit. awk already matches lines 🙂  `./jdk-wrapper.sh ./mvnw -v | awk '/^Maven home:/ { print $3 }' `

Reply:
Yeah, I just cribbed it and didn't try to make it better; just validated it works. Fixed!

FYI. This will all be deleted once we are able to release from the private Jenkins instance.